MekHQ/userdata/data/universe/ranks.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<rankSystems version="0.50.1-SNAPSHOT">
+<rankSystems version="0.50.2-SNAPSHOT">
```

Comment:
Shouldn't this be 50.02

Reply:
So, yes, but not according to our versioning. I think Tex might need to take a look at that.

Reply:
I just fixed it :)
![image](https://github.com/user-attachments/assets/4cc7e7bf-7e8b-4aba-9a05-b0460eb55a1c)

Reply:
Perfect! I'll get this PR updated when I'm home :)